### ISSUE TITLE
Create the crmusr in CRM?

### ISSUE THREAD

**Issue by stevechin1986** [Microsoft/CRM-Performance-Toolkit#8]
Hello.

I have a question In regards to the user creation pre-requisite for this tool:

The Microsoft Dynamics CRM 2015 Performance Toolkit Documentation lists a cmd script to create the Active Directory Users. It's not stated anywhere that these users should be created in CRM. Is that implied? 

Or does the DbPopulator Tool take care of that?
(My fear is that if I create these users... then do I have to assign roles... etc etc).

Thanks!

**Comment by jeffreyzhzhang** [comment]
Yes, the users have to be created in CRM and this script - <toolkit install dir>\CRMToolkit\Binaries\Scripts\ps\DataPopulation_320UserBaseline.ps1, will take care of user creation and assigning user roles. Specifically this is the code in the script for creating users - PopulateData $dbPopExe $dataFilePath\OrgDataFiles\OrgStructure.xml $configProperties.
